assets/openobserve/search.py

```
@@ -0,0 +1,45 @@
+openobserve_host = "http://localhost:5080"
+openobserve_url = openobserve_host + "/api/" + org + "/_search"
+
+query = sys.argv[1]
+
+pagesize = 1000
+
+i = 0
+while True:
+    data = {
+            "query": {
+                "sql": f"SELECT id FROM {stream} WHERE {query}",
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)